tests/lib/extractors/test_review_model.py

```diff
@@ -0,0 +1,167 @@
+"""Tests for the text metadata extractors."""
```

Comment:
It's hard to tell what's actually being tested, since there's so much mocking/patching. Is it possible to create more "complete" test cases, and test functionality with less mocking?